airflow/models/dagbag.py

```diff
@@ -600,7 +600,12 @@ def _serialize_dag_capturing_errors(dag, session):
             except OperationalError:
                 raise
             except Exception:
-                self.log.exception("Failed to write serialized DAG: %s", dag.full_filepath)
+                self.log.exception("Failed to write serialized DAGs in: %s", dag.fileloc)
```

Comment:
```suggestion
                self.log.exception("Failed to write serialized DAG %s in: %s", dag.dag_id, dag.fileloc)
```

@kaxil, this is probably even better, since it is a failure at the DAG level at this point.